instrumentation/android-log/library/src/main/java/io/opentelemetry/instrumentation/library/log/AndroidLogSubstitutions.java

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.library.log;
+
+import static io.opentelemetry.instrumentation.library.log.internal.LogRecordBuilderCreator.createLogRecordBuilder;
+import static io.opentelemetry.instrumentation.library.log.internal.LogRecordBuilderCreator.getTypeName;
+import static io.opentelemetry.instrumentation.library.log.internal.LogRecordBuilderCreator.printStacktrace;
+
+import android.util.Log;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.logs.Severity;
+
+public class AndroidLogSubstitutions {
+
+    public static AttributeKey<String> TAG_KEY = AttributeKey.stringKey("android.log.tag");
+
+    public static AttributeKey<String> EXCEPTION_STACKTRACE =
+            AttributeKey.stringKey("exception.stacktrace");
+
+    public static AttributeKey<String> EXCEPTION_TYPE = AttributeKey.stringKey("exception.type");
```

Comment:
These two attributes already exist in the semantic conventions. We should reference those constants, rather than redefining them there. See the `ExceptionAttributes` class.